config/manager/manager.yaml

```diff
@@ -72,22 +74,9 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
-        #resources:
-        #  limits:
-        #    cpu: 1000m
-        #    memory: 1000Mi
-        #  requests:
-        #    cpu: 1000m
-        #    memory: 1000Mi
-        #volumeMounts:
-        #  - mountPath: /opt/splunk/appframework/
-        #    name: app-staging
```

Comment:
Nit: Having the `volumeMounts:` may be handy for the customer, as we are recommending it.

Reply:
But it's commented out? Are we recommending a pvc for app-framework?

Reply:
I think @vivekr-splunk commented on it for now. Ideally, it should have gone to 2.0.0 directly. Even on 2. x, it will be commented, being it is a recommendation in the case of app framework. Also, it needs the customer to create the PVC first, before uncommenting this.

Reply:
If its going to be commented out, I believe it doesn't get exported to `splunk-operator-install.yaml`? Maybe it should be a part of the documentation for app-framework such that they can edit the operator deployment as well as create a pvc i.e we could provide a `kubectl patch` command for the deployment as well as a simple yaml for pvc?

Reply:
we do not need this in develop for now. when we merge appframework branch it will be enabled.